rules/eslint/best-practices/eslint.js

```diff
@@ -9,94 +9,100 @@ module.exports = {
     // specify the maximum cyclomatic complexity allowed in a program
     "complexity": [0, 11],
     // require return statements to either always or never specify values
-    "consistent-return": 2,
+    "consistent-return": 0,
     // specify curly brace conventions for all control statements
-    "curly": [2, "all"],
+    "curly": [0, "all"],
```

Comment:
All of your comments are about the Eslint's ruleset which is just a mirror of this file. I'm simply exposing what ESLint will use.

https://github.com/eslint/eslint/blob/master/conf/eslint.json